### PR TITLE
Use magnificent, the missing tests

### LINKED ISSUE #95: Use magnificent
To make sure all microservices have the same understanding of (write) permissions.

### PR DESCRIPTION
**Pull request by prayerslayer**
Fixes #95 
- Add tests
- Put `require-` things into own module

### CODE REVIEW

File: src/org/zalando/stups/pierone/sql.clj [hunk 1]
```diff
@@ -7,7 +7,7 @@
 (def default-db-configuration
   {:db-classname   "org.postgresql.Driver"
    :db-subprotocol "postgresql"
-   :db-subname     "//localhost:5432/pierone"
+   :db-subname     "//192.168.99.100:5432/pierone"
```
Comment (hjacobs) [inline src/org/zalando/stups/pierone/sql.clj]: ohoh

Comment (prayerslayer) [inline src/org/zalando/stups/pierone/sql.clj]: Docker...

**Comment by prayerslayer** [comment]
👍

**Comment by prayerslayer** [comment]
Minimum is now 2

**Comment by dryewo** [comment]
👍

**Pull request merged**